packages/ckeditor5-ui/tests/toolbar/toolbarview.js

```diff
@@ -118,6 +119,17 @@ describe( 'ToolbarView', () => {
 			expect( view.itemsView.element.classList.contains( 'ck-toolbar__items' ) ).to.true;
 		} );
 
+		it( 'should include ck-toolbar_floating class if there are shouldGroupWhenFull and isFloating options set to true', () => {
+			const viewWithOptions = new ToolbarView( locale, {
+				shouldGroupWhenFull: true,
```

Comment:
What if `shouldGroupWhenFull` is `false`? Would this work?

Reply:
I've added more tests about it